Libraries/Azure.psm1

```diff
@@ -1248,17 +1289,17 @@ Function Create-AllResourceGroupDeployments($SetupTypeData, $TestCaseData, $Dist
 				} else {
 					Add-Content -Value "$($indents[5])^linuxConfiguration^:" -Path $jsonFile
 					Add-Content -Value "$($indents[5]){" -Path $jsonFile
-						Add-Content -Value "$($indents[6])^disablePasswordAuthentication^:true," -Path $jsonFile
-						Add-Content -Value "$($indents[6])^ssh^:" -Path $jsonFile
-						Add-Content -Value "$($indents[6]){" -Path $jsonFile
-							Add-Content -Value "$($indents[7])^publicKeys^:" -Path $jsonFile
-							Add-Content -Value "$($indents[7])[" -Path $jsonFile
-								Add-Content -Value "$($indents[8]){" -Path $jsonFile
-									Add-Content -Value "$($indents[9])^path^:^$sshPath^," -Path $jsonFile
-									Add-Content -Value "$($indents[9])^keyData^:^$sshKeyData^" -Path $jsonFile
-								Add-Content -Value "$($indents[8])}" -Path $jsonFile
-							Add-Content -Value "$($indents[7])]" -Path $jsonFile
-						Add-Content -Value "$($indents[6])}" -Path $jsonFile
+					Add-Content -Value "$($indents[6])^disablePasswordAuthentication^:true," -Path $jsonFile
```

Comment:
the indent is make the json file more readable.

Reply:
indent will be controlled by $indents[int number], right? does it related to the indenting of the source code here itself of preparing the template file?